pom.xml

```diff
@@ -16,12 +16,12 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
         <spring-boot.version>2.6.8</spring-boot.version>
-        <jackson-databind-nullable.version>0.2.2</jackson-databind-nullable.version>
+        <jackson-databind-nullable.version>0.2.4</jackson-databind-nullable.version>
         <org.mapstruct.version>1.4.2.Final</org.mapstruct.version>
         <org.projectlombok.version>1.18.20</org.projectlombok.version>
         <org.projectlombok.e2e-checks.version>1.18.20</org.projectlombok.e2e-checks.version>
         <git.revision>latest</git.revision>
-        <kafka-clients.version>3.2.0</kafka-clients.version>
+        <kafka-clients.version>3.2.3</kafka-clients.version>
```

Comment:
question: Should this be the same version as in `kafka-ui-e2e-checks/pom.xml`?

Reply:
yep